.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+-   repo: https://github.com/awslabs/git-secrets
+    rev: 80230afa8c8bdeac766a0fece36f95ffaa0be778
```

Comment:
i believe you only need first 7 numbers of the SHA?  Also wondering why you don't set it to a released tag?

Reply:
Hi,

Actually, the most recent released tag (1.3.0) does not support the pre_commit_hook structure. It was introduced after this release (this commit: b4a3208)